examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -107,26 +73,29 @@ public static void main(String[] args) throws Exception {
             System.exit(0);
         }
 
-        HelloWorldClientTls client;
+        // If only defaults are necessary, you can use TlsChannelCredentials.create() instead of
+        // interacting with the Builder.
```

Comment:
Just a suggestion: you can illustrate the `TlsChannelCredentials.create()` usage by

- declaring a TlsChannelCredentials var instead of the builder
- for case 5 and 3 use the builder and `build()` to assign to the creds var
- for case 2 just use TlsChannelCredentials.create()